packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessabilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,119 @@
+import { Notice } from "@cerner/terra-docs";
+import Whitespace from "../../common/layout-helpers/Whitespace";
+import { Badge } from 'terra-icon/package.json?dev-site-package';
+
+<!-- Examples -->
+import DecorativeIconExample from './example/DecorativeIconExample?dev-site-example';
+import GroupOfIconsExample from './example/GroupOfIconsExample?dev-site-example';
+import InformativeIconExamples from './example/InformativeIconExamples?dev-site-example';
+import FunctionalIconExample from './example/FunctionalIconExample?dev-site-example';
+
+<!-- Icons used in the examples, but also I want to use them in this doc. -->
+import IconManufacturer from 'terra-icon/lib/icon/IconManufacturer';
+import IconAnnouncement from 'terra-icon/lib/icon/IconAnnouncement';
+import IconTile from 'terra-icon/lib/icon/IconTile';
+import IconList from 'terra-icon/lib/icon/IconList';
+import IconTreemap from 'terra-icon/lib/icon/IconTreemap';
+
+<Badge />
+
+# Accessibility Guide for Terra Icon
+
+<Notice variant="important" ariaLevel="2">
+
+Accessibility of icons is important because icons are ubiquitous. They can convey visual meaning or be used as a decorative or supportive element in a UI design and are an essential tool for creating intuitive user experiences that are appropriate for all languages. However, **if implemented improperly or without alternative content, they will not understood everyone**.
```

Comment:
Did you mean "they will not **_be_** understood **_by_** everyone"?

Reply:
Fixed in 8be2421ad11674691395463c2e5518e0dfc27aae.